package.json

```diff
@@ -80,24 +80,33 @@
   },
   "homepage": "https://github.com/department-of-veterans-affairs/vets-website#readme",
   "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-decorators": "^7.0.0",
```

Comment:
Didn't know we used these.

Reply:
We don't. It got added automatically because it was in the stage2 preset, but I'll remove it.